plugins/performance-lab/readme.txt

```diff
@@ -11,7 +11,19 @@ Performance plugin from the WordPress Performance Team, which is a collection of
 
 == Description ==
 
-The Performance Lab plugin is a collection of features focused on enhancing performance of your site, most of which should eventually be merged into WordPress core. The plugin allows to individually enable and test the features to get their benefits before they become available in WordPress core, and to provide feedback to further improve the solutions.
+The Performance Lab plugin is a collection of features focused on enhancing performance of your site, most of which should eventually be merged into WordPress core. The plugin facilitates the discovery and activation of the individual performance feature plugins which the performance team is developing. In this way you can test the features to get their benefits before they become available in WordPress core. You can also play an important role by providing feedback to further improve the solutions. 
+
+The feature plugins which are currently featured by this plugin are:
+
+* [Embed Optimizer](https://wordpress.org/plugins/embed-optimizer/)
+* [Enhanced Responsive Images](https://wordpress.org/plugins/auto-sizes/)
+* [Image Placeholders](https://wordpress.org/plugins/dominant-color-images/)
+* [Image Prioritizer](https://wordpress.org/plugins/image-prioritizer/)
+* [Modern Image Formats](https://wordpress.org/plugins/webp-uploads/)
+* [Performant Translations](https://wordpress.org/plugins/performant-translations/)
+* [Speculative Loading](https://wordpress.org/plugins/speculation-rules/)
```

Comment:
I think we should match the order of these with how they're displayed in the admin, and possibly include the "(experimental)" bit as well where relevant. In other words:
* Sort by slug.
* Put experimental plugins after non-experimental ones.
 
See https://github.com/WordPress/performance/blob/trunk/plugins/performance-lab/load.php#L88.

We could revisit the sorting by slug, since it makes a bit more sense to sort by the user-facing name, but IMO that's a bit of a detail that could be separately explored. My main point here is to follow a consistent approach to sorting.

Reply:
+ to sorting by user facing name vs slug

Reply:
FWIW I fixed the sorting in #1098. It sorts by user-facing name and puts experimental ones last (as we already do today). Contains a few other code quality improvements too.  I should probably split that up into its own PR

Reply:
Makes sense, in that case I think we should also update the PL admin screen logic to sort by user-facing name once the plugin information has been fetched from .org.

Reply:
@swissspidy I've cherry-picked your commit to sort by name.

I've put the experimental plugins at the end in the readme.